src/auxil/auxil/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "gfx-auxil"
-version = "0.6.0"
+version = "0.6.1"
```

Comment:
this isn't backwards compatible, should be 0.7.0 now

Reply:
Oops, nice catch :)